fees/strata-markets/index.ts

```
@@ -17,72 +26,116 @@ const fetch = async (options: FetchOptions) => {
+  const vaultAddresses = [JRT, SRT];
+
+  const [depositLogs, withdrawLogs] = await Promise.all([
+    options.getLogs({ targets: vaultAddresses, eventAbi: DEPOSIT_EVENT }),
+    options.getLogs({ targets: vaultAddresses, eventAbi: WITHDRAW_EVENT }),
+  ]);
 
-  // Calculate performance fees from gains (only if there are gains)
-  if (navGrowth > 0 && reserveBps > 0) {
-    // Reserve gets a portion of gains: gain * reserveBps / 1e18
-    performanceFees = Number(navGrowth * reserveBps / 1e18);
+  depositLogs.forEach((log: any) => totalDeposits += Number(log.assets));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)